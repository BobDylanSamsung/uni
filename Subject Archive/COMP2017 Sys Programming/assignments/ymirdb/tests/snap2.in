set a 1 2 3
snapshot
set b 1 a 2
snapshot
set c a b
snapshot
list snapshots
list entries
checkout 1
list snapshots
list entries
set b 1 2 3
del a
list entries
checkout 2
list entries
del b
checkout 3
list entries
bye
