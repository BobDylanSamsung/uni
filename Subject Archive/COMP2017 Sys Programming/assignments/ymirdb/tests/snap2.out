ok

saved as snapshot 1

ok

saved as snapshot 2

ok

saved as snapshot 3

3
2
1

c [a b]
b [1 a 2]
a [1 2 3]

ok

3
2
1

a [1 2 3]

ok

ok

b [1 2 3]

ok

b [1 a 2]
a [1 2 3]

ok

ok

c [a b]
b [1 a 2]
a [1 2 3]

bye
